code/game/objects/items/devices/radio/radio_vr.dm

```diff
@@ -0,0 +1,127 @@
+//Pathfinder's Subspace Radio
+/obj/item/device/subspaceradio
+	name = "subspace radio"
+	desc = "A powerful new radio recently gifted to Nanotrasen from KHI, this communications device has the ability to send and recieve transmissions from anywhere."
+	icon = 'icons/vore/custom_items_vr.dmi'
+	icon_state = "radiopack"
+	item_state = "parachute"
+	slot_flags = SLOT_BACK
+	force = 5
+	throwforce = 6
+	preserve_item = 1
+	w_class = ITEMSIZE_LARGE
+	action_button_name = "Remove/Replace Headset"
+
+	var/obj/item/device/radio/subspaceradio/linked/headset
+
+/obj/item/device/subspaceradio/New() //starts without a cell for rnd
+	..()
+	if(ispath(headset))
+		headset = new headset(src, src)
+	else
+		headset = new(src, src)
+
+/obj/item/device/subspaceradio/Destroy()
+	. = ..()
+	QDEL_NULL(headset)
+
+/obj/item/device/subspaceradio/ui_action_click()
+	toggle_headset()
+
+/obj/item/device/subspaceradio/attack_hand(mob/user)
+	if(loc == user)
+		toggle_headset()
+	else
+		..()
+
+/obj/item/device/subspaceradio/MouseDrop()
+	if(ismob(src.loc))
```

Comment:
Implied `src`? :thinking:

Is that something that's even reviewed for in Polaris code?

Reply:
This is copy-pasted straight from defib code, which is older than Polaris.

Reply:
Defibs are not older than Polaris. Defibs were added to Bay after I added them to Polaris, and Bay never had /tg/ defibs, nor are theirs a port of them.

Reply:
Would rather you remove the explicit srcs unless there's a need for clarity (e.g. src and a higher scope having the same var name), generally bad style.

Reply:
I am not sure how to rewrite it to use less srcs.